DANS-KNAW/dans-dataverse-client-lib
# DD-999: part 3 - new methods called by dd-ingest-flow

Fixes DD-999: part 3 - new methods called by dd-ingest-flow

# Description of changes
* new SwordApi to delete files
* new AccessRequestsApi to dis/en-able
* new AbstractIdApi: URL construction with persistent id or normal id from DatasetApi
* more verbose logging of DataverseException, frequently an HTML page to wade through (search for lines with attribute method to figure out the sent URL) but in case of the SwordApi authorization quite concise and informative


# How to test
As in https://github.com/DANS-KNAW/dd-ingest-flow/pull/20
but make sure one `<ddm:available>` is in the future and one `bag-info.txt` contains `Is-Version-Of: urn:uuid:...`
* [x] SwordApi.deleteFile fixed 
* [x] AccessRequestsApi (dis/enable) fixed

# Related PRs 
* [ ] dd-ingest-flow

# Notify
@DANS-KNAW/dataversedans
